### DISCUSSION TITLE
Single window for documentaries, (non race videos)

### DISCUSSION THREAD

**Discussion by argrithmag** [f1multiviewer/issue-tracker#527] [Feature requests]
I just went on a binge of documentaries at a friend's house.  We hooked my laptop up to his TV as a second monitor.  

First thing I noticed was that every time I opened a new documentary or video, it opened in its own window.  I then had to close the previous video, drag the new one over to the TV, maximize it, turn on volume, and turn on subtitles (when available).

It would be great if for non-race videos, it could start new videos in the same window with the same setting.

**Comment by jeppeolesen** [comment]
Duplicate of #394
